lib/utils/domFns.es6

```diff
@@ -127,21 +127,24 @@ export function getTouchIdentifier(e: MouseTouchEvent): ?number {
 // User-select Hacks:
 //
 // Useful for preventing blue highlights all over everything when dragging.
-const userSelectPrefix = getPrefix('user-select');
-const userSelect = browserPrefixToStyle('user-select', userSelectPrefix);
-const userSelectStyle = `;${userSelect}: none;`;
-const userSelectReplaceRegExp = new RegExp(`;?${userSelect}: none;`); // leading ; not present on IE
 
 // Note we're passing `document` b/c we could be iframed
-export function addUserSelectStyles(body: HTMLElement) {
-  const style = body.getAttribute('style') || '';
-  if (userSelectReplaceRegExp.test(style)) return; // don't add twice
-  body.setAttribute('style', style + userSelectStyle);
+export function addUserSelectStyles(doc: Document) {
+  let styleEl = doc.getElementById('react-draggable-style-el');
+  if (!styleEl) {
+    styleEl = doc.createElement('style');
+    styleEl.type = 'text/css';
+    styleEl.id = 'react-draggable-style-el';
+    styleEl.innerHTML = '.react-draggable-transparent-selection *::-moz-selection {background: transparent;}\n';
+    styleEl.innerHTML += '.react-draggable-transparent-selection *::selection {background: transparent;}\n';
+    doc.getElementsByTagName('head')[0].appendChild(styleEl);
+  }
+  doc.body.classList.add('react-draggable-transparent-selection'); 
```

Comment:
Unfortunately we can't count on this existing (although it does in every modern browser IE11+). My preference would be to munge the string so we maintain IE9 support.